### PR TITLE
Account Verification: Added `VerifyAccount` UI.

### PR DESCRIPTION
**Pull request by KH-Amir-TN**
+ Integrated with the `VerifyAccountController#create` API.

## Description

Added `VerifyAccount` UI and integrated it with `VerifyAccountController#create` API.
## Testing Steps

> 1. Pull the code.
> 2. Install the dependencies `bundle install && npm|yarn install`.
> 3. Clean the previous assets build by running `rm app/assets/builds/*` (This won't remove .keep since it's hidden).
> 4. Clean the database and tmp files for a better isolation by running `rails tmp:clear && rails db:schema:cache:clear && rails db:drop && rails db:create && rails db:migrate:with_data`
> 5. Run the linter and specs `bundle exec rubocop --parallel && bundle exec rspec && npx eslint app/javascript/* --ext .jsx,.js`
> 6. Run `./bin/dev` to run the assets builders processes and the Puma server all at once.

## Screenshots (if appropriate):

image

### CODE REVIEW

**Comment by KH-Amir-TN** [comment]
# DEPENDS ON #3969

**Comment by scouillard** [comment]
Putting the Button component fix the width for some reason.

<div>
  <Button />
</div>

**Comment by KH-Amir-TN** [comment]
# REBASED ON bb3ebebd0b26c0b9db4264f76352cea455d8e368

File: app/javascript/components/users/account_activation/VerifyAccount.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        <h1>{ t('account_activation_page.title') }</h1>
+      </Row>
+      <Row>
+        <Col md={8} className="mx-auto">
+          <Card className="p-5 border-0 shadow-sm">
+            <Card.Title className="text-center pb-2"> <InformationCircleIcon className="hi-xl" /> </Card.Title>
+            <pre className="text-muted">
+              { t('account_activation_page.message') }
+            </pre>
+            <Button onClick={createActivationLinkAPI.mutate} disabled={createActivationLinkAPI.isLoading}>
```
Comment (farhatahmad) [inline app/javascript/components/users/account_activation/VerifyAccount.jsx:28]: This should be branded

File: public/locales/en.json [hunk 2]
```diff
@@ -324,5 +325,10 @@
   "not_found_error_page": {
     "title": "404",
     "message": "The resource you are trying to access cannot be found.\nMaybe you were looking for something else?"
+  },
+  "account_activation_page": {
+    "title": "Account Activation",
+    "message": "Your account has not been verified yet.\nIn order to use Greenlight full features you are kindly invited to verify your account.\nIf you haven't received an activation link or if you are having a problem using it,\nplease click the resend button below to get a new link.\nYou can still update your profile information in case you used the wrong email address.",
```
Comment (farhatahmad) [inline public/locales/en.json:331]: Let's update this to match what we agreed in the last meeting

File: public/locales/en.json [hunk 3]
```diff
@@ -324,5 +325,10 @@
...
+    "resend_btn_lbl": "Resend verification"
```
Comment (farhatahmad) [inline public/locales/en.json:332]: `Resend Verification`

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**